nncf/experimental/onnx/algorithms/quantization/default_quantization.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
Why is this file called "default_quantization"?

Reply:
This is a standard name of such files in other NNCF QAT backends. This file reflects the default scheme of metatypes mapping the quantization

Reply:
I also don't like the name. I prefer `quantization_metatypes.py` or `metatypes_to_quantization_trai.py`
Three options here:
1) Leave as it is.
2) Rename from `default_quantization.py` only for **ONNX**.   - I favor this point.
3) Rename from `default_quantization.py` for **all backends**.

What do you think?

Reply:
This is a file describing the *default* quantization traits for operations which have no associated information coming from the HW config due to one or the other reason. The current naming is proper.

Reply:
For now, at least we could agree on renaming this for ONNX.

Reply:
So, @vshampor said the name is correct while @AlexKoff88 vice versa.
Let's finalize the name

Reply:
Decided to save the name 'deafault_quantzation.py', if you have any confusion, please reopen the conversation.